integration/docker/docker.go

```diff
@@ -34,10 +34,7 @@ const (
 	DebianImage = "debian"
 
 	// FedoraImage is the fedora image
-	FedoraImage = "fedora"
-
-	// CentosImage is the centos image
-	CentosImage = "centos"
+        FedoraImage = "fedora"
```

Comment:
Is this an intentional change - unclear to me (might have been clearer if it was in its own commit ;-) )